data_analysis/heart_rate_generator.py

```diff
@@ -5,10 +5,8 @@
 import os
 
 pwd = '/home/'
-print 'Loading model'
```

Comment:
Maybe finally introduce your  beloved logger for this sort of thing. I think it might be still useufll to know this... Especially when we swap the models in and out...